backend/process_miner/logs_process_miner.py

```diff
@@ -0,0 +1,141 @@
+"""
+Module used for importing tagged logs into heuristic miner
+"""
+import os
+
+from pm4py.objects.log.adapters.pandas import csv_import_adapter
+
+from pm4py.objects.conversion.log import factory as conversion_factory
+
+from pm4py.algo.discovery.heuristics import algorithm as heuristics_miner
+from pm4py.visualization.heuristics_net import visualizer as hn_vis
+
+from pm4py.visualization.dfg import factory as dfg_vis_factory
+from pm4py.algo.discovery.dfg import factory as dfg_factory
+
+from pm4py.util import constants
+
+from pm4py.algo.filtering.pandas.attributes import attributes_filter
+
+POSSIBLE_APPROACHES = {"embedded": "embedded", "redirect": "redirect",
+                       "OAuth": "OAuth", "all": "all",
+                       "not available": "not available"}
```

Comment:
This should be a list as the str -> str mapping doesn't serve any purpose if both strings are identical